samples/javascript/KafkaOutput/function.json_

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Rename to function.json? Inconsistent with other folders

Reply:
Done